static/payment.js

```diff
@@ -0,0 +1,60 @@
+function setupPayment(stripeKey) {
+    var stripe = Stripe(stripeKey);
+    var elements = stripe.elements({locale: "sv"});
+
+    var card = elements.create('card', {
+        hidePostalCode: true,
+        style: {
+            base: {
+                iconColor: '#666EE8',
+                    <!-- color: '#31325F', -->
```

Comment:
I'll fix the comment. I don't know if it's possible to use class selectors instead.

I think I will leave the functionality as is for the moment and then tidy that up at a later point. This PR is huge enough as it is.